job_spec/INSAR_MULTI_BURST.yml

```diff
@@ -11,7 +11,7 @@ INSAR_MULTI_BURST:
         maxItems: 50
```

Comment:
50 seems like a very large number that could tax our systems. What about setting it to 30? This would be roughly the size of a standard SLC.

Reply:
Yeah I think @asjohnston-asf had thrown out 50 as a ballpark number, but 30 seems fine too. We can always increase later.